docs/data/migration/migration-data-grid-v5/migration-data-grid-v5.md

```diff
@@ -65,7 +65,7 @@ The minimum supported Node.js version has been changed from 12.0.0 to 14.0.0, si
   The old behavior can be restored by using `apiRef.current.stopRowEditMode({ ignoreModifications: true })` or `apiRef.current.stopCellEditMode({ ignoreModifications: true })`.
 - The `onColumnVisibilityChange` prop was removed. Use `onColumnVisibilityModelChange` instead.
```

Comment:
I see a lot of cases (especially in data grid side) of `was removed` and only some `has been`.
I'm not sure which form is more correct.
- Prop has been removed. Please update usage.
- Prop was removed. Please update usage.

Maybe @samuelsycamore could confirm? 🤔

Reply:
Thanks for clarifying. 😉 
Will continue using "_has been removed_". 👍